airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/S3Destination.java

```diff
@@ -33,7 +41,15 @@ public static void main(final String[] args) throws Exception {
   @Override
   public AirbyteConnectionStatus check(final JsonNode config) {
     try {
-      attemptS3WriteAndDelete(S3DestinationConfig.getS3DestinationConfig(config), config.get("s3_bucket_path").asText());
```

Comment:
should we just replace all usages of this method, or is it possible that the staging warehouse destinations do actually need those extra permissions? E.g. https://github.com/airbytehq/airbyte/blob/ae2a87ee0563e8d54b341422ceb3ecef09afc69d/airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeCopyS3Destination.java#L40

Reply:
I think it would depend on the use case, i.e. how the warehouse uses the S3 or other staging bucket. Since there is no complaint from the community about any permission issue for those destinations, we have not looked into them yet.